src/List/__tests__/item.spec.js

```diff
@@ -0,0 +1,75 @@
+import { createElement } from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import { Item } from '../index.js';
+
+configure({ adapter: new Adapter() });
+
+const classes = {
+    root: 'abc'
+};
+
+test('renders a div by default', () => {
+    const props = { classes, item: 'a' };
+    const wrapper = shallow(<Item {...props} />).dive();
+
+    expect(wrapper.type()).toEqual('div');
+    expect(wrapper.prop('className')).toEqual(classes.root);
```

Comment:
This assertion isn't related to the described test case. I'd expect `renders a div by default` to keep passing even if we change how we're styling the component.